contracts/EVMScriptFactories/AddAllowedRecipient.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.4;
+
+import "../TrustedCaller.sol";
+import "../AllowedRecipientsRegistry.sol";
+import "../libraries/EVMScriptCreator.sol";
+import "../interfaces/IEVMScriptFactory.sol";
+
+/// @author psirex, zuzueeka
+/// @notice Creates EVMScript to add new allowed recipient address to AllowedRecipientsRegistry
+contract AddAllowedRecipient is TrustedCaller, IEVMScriptFactory {
+    // -------------
+    // ERRORS
+    // -------------
+    string private constant ERROR_ALLOWED_RECIPIENT_ALREADY_ADDED =
+        "ALLOWED_RECIPIENT_ALREADY_ADDED";
+
+    // -------------
+    // VARIABLES
+    // -------------
+
+    /// @notice Address of AllowedRecipientsRegistry
+    AllowedRecipientsRegistry public immutable allowedRecipientsRegistry;
```

Comment:
Let's not go with `immutable` here? Would be extremely painful to verify different deployments otherwise.